specification/resources/resource-manager/Microsoft.Authorization/2016-12-01/policyDefinitions.json

```diff
@@ -137,6 +137,38 @@
         }
       }
     },
+    "/providers/Microsoft.Authorization/policydefinitions/{policyDefinitionName}": {
```

Comment:
was this api missing from the spec? or is it newly added to the existing api version?

Reply:
this was missing from the spec